web/app/js/components/Navigation.jsx

```diff
@@ -245,33 +241,20 @@ class NavigationBase extends React.Component {
               <ListItemIcon><LibraryBooksIcon /></ListItemIcon>
               <ListItemText primary="Documentation" />
             </ListItem>
+            <ListItem component="a" href="https://lists.cncf.io/g/cncf-linkerd-users" target="_blank">
```

Comment:
When you click on one of these links the button highlight remains and it looks a little strange. 
![screen shot 2019-02-21 at 3 50 09 pm](https://user-images.githubusercontent.com/549258/53209769-684fe380-35f0-11e9-8399-066b1a5b9913.png)

We could remove this by adding:

In styles:
```
    helpMenuItem: {
      outline: "none",
    },
```

And here, 
```
<ListItem component="a" href="https://linkerd.io/2/overview/" target="_blank" className={classes.navMenuItem}>
```

Reply:
Good point - now that these links are surfaced it's more jarring. Fixing!